lib/DDG/Spice/Shares.pm

```diff
@@ -0,0 +1,20 @@
+package DDG::Spice::Shares;
+# ABSTRACT: Stock quote ticker
+
+use strict;
+use DDG::Spice;
+
+triggers startend => 'stock quote for', 'stock quote', 'stock price of', 'stock price', 'share price', 'stocks', 'shares', 'share price for', 'share price of';
+
+spice to => 'https://og6c1r82zb.execute-api.us-east-1.amazonaws.com/v1/ddg?term=$1';
```

Comment:
Hi, @moollaza 

Yes we created this endpoint just for DDG, it is simply a AWS API-Gateway endpoint triggering a Lambda Function that in turn uses our internal APIs.  We decided to keep it separate from the api that serves our site `https://api.stockflare.com` just for simplicity sake.

We can easily put it on a custom domain using our certificate if you prefer, something like ddg.stockflare.com